charlesinto/iReporter
# #162532134 POST A NEW RECORD

#### What does this PR do?

- post new incident

#### Description of Task to be completed?
- implement jwt authentication
- verify user token
- user should be able to post new report

#### How should this be manually tested?
- Clone the repo - `git clone  git repo address`
- Enter into the project directory - `cd repo folder`
- switch to this branch - `PR branch`
-  Start  the frontend.
- Navigate to `endpoint`
- Open `Postman`
- Grab your token from the application tab in Developers tab
- On postman add the token in headers
- - On `postman` Send a `POST` request to `http://localhost:7000/api/v1/red-flags/`
- You should receive a successful response

#### Any background context you want to provide?
- the POST endpoint response should be structured like this- 



`

#### What are the relevant pivotal tracker stories?
[#162532134 ](https://www.pivotaltracker.com/story/show/162532134 )

#### Screenshots (if appropriate)

![image](https://user-images.githubusercontent.com/26615085/49704114-288f7780-fc0f-11e8-9c71-020b0250619a.png)


#### Questions:
N/A
